mapper.go

```diff
@@ -24,10 +24,11 @@ import (
 )
 
 var (
-	identifierRE = `[a-zA-Z_][a-zA-Z0-9_]+`
-	metricLineRE = regexp.MustCompile(`^(\*\.|` + identifierRE + `\.)+(\*|` + identifierRE + `)$`)
-	labelLineRE  = regexp.MustCompile(`^(` + identifierRE + `)\s*=\s*"(.*)"$`)
-	metricNameRE = regexp.MustCompile(`^` + identifierRE + `$`)
+	identifierRE      = `[a-zA-Z_][a-zA-Z0-9_]*`
+	metricLineRE      = regexp.MustCompile(`^(\*\.|` + identifierRE + `\.)+(\*|` + identifierRE + `)$`)
```

Comment:
I wasn't able to find a reliable source of what characters are allowed in Graphite metric names, but some posts suggested that even dashes or parentheses might be legal?